<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineConfigure</class>
 <widget class="QDialog" name="MachineConfigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(27, 150, 208);
}
QLabel
{
font-size:26px;
color: rgb(255, 255, 255);
}
QPushButton{
border-radius:12px;
background-color: rgb(80, 80, 80);
font-size:22px;
color:rgb(255, 255, 0);
}
QPushButton::checked{
color:red;
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}

QToolButton{
font-size:26px;
font:bold;
color:blue;
background-color:transparent;
border:noborder;
}
QToolButton::checked{
color:red;
}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font-size:28px;</string>
   </property>
   <property name="text">
    <string>机号匹配设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设备ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>起始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>到期时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_machineid">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_starttime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_endtime">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_read">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>61</height>
       </size>
      </property>
      <property name="text">
       <string>读取主板信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_configure">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>61</height>
       </size>
      </property>
      <property name="text">
       <string>配置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>61</height>
       </size>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>61</height>
       </size>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
